buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -76,6 +76,10 @@ object Versions {
     internal const val TIMBER = "5.0.1"
     internal const val TURBINE = "0.12.1"
     internal const val WORK = "2.7.1"
+    internal const val ANDROIDX_TEST = "1.4.0"
+    internal const val BASE_PROFILE = "1.2.0"
+    internal const val MACRO_BENCHMARK = "1.1.0"
+    internal const val ANDROIDX_UI_AUTOMATOR = "2.2.0"
```

Comment:
Could we sort them alphabetically?

Reply:
Done: [3110edd](https://github.com/GetStream/stream-chat-android/pull/4610/commits/3110eddc3a7e82fb85397a13d91d77662a26a226)